client/src/pages/OnBoarding/smallComponents/AvailableHoursButton.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react";
+import { TextField, MenuItem } from "@material-ui/core";
+
+const timeSlots = [];
+for (let i = 0; i <= 24; i++) {
```

Comment:
```suggestion
for (let i = 0; i < 24; i++) {
```
Right now this is causing the times to go past midnight, note that we should also append 24:00 at the end of the array when finished